web/src/beta/components/fields/RangeField/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { useCallback, useEffect, useState } from "react";
+
+import Property from "@reearth/beta/components/fields";
+import RangeSlider, { Props as RangeProps } from "@reearth/beta/components/RangeSlider";
+
+export type Props = {
+  name?: string;
+  description?: string;
+} & RangeProps;
+
+const RangeField: React.FC<Props> = ({ name, description, value, onChange, ...args }: Props) => {
+  const [internalState, setInternalState] = useState<number[] | undefined>(value ?? [0, 30]);
```

Comment:
RangeField isn't always going to be used for tile's zoom levels, so the [0, 30] isn't appropriate here.
If `args` includes min/max, you could destructure those from the props and use those as a starting value.

Reply:
I also remove this value